### ISSUE TITLE
add avahi/mDNS support

### ISSUE THREAD

**Issue by ccrisan** [ccrisan/motionPie#57]

**Comment by CharlJKruger** [comment]
regarding dynamic dns's, would it be possible to build in a feature that could like wget a custom url at a specific time interval to update the ip address. Then I would be able to set up a ddns script to update motionpie's ip and view it externally without the need for a static ip?

Thanks for the awesome product and let me know.

**Comment by ccrisan** [comment]
Yes, this could be implemented; I'll think of a way to make this available.

**Comment by CharlJKruger** [comment]
great stuff!

**Comment by NizzeHulth** [comment]
I have full respect for you're workload, but can you give an indication when we can see this feature?
